pkg/volume/azure_file/azure_util.go

```diff
@@ -138,5 +149,13 @@ func appendDefaultMountOptions(mountOptions []string, fsGroup *int64) []string {
 	if !gidFlag && fsGroup != nil {
 		allMountOptions = append(allMountOptions, fmt.Sprintf("%s=%d", gid, *fsGroup))
 	}
+
+	if !actimeoFlag {
+		allMountOptions = append(allMountOptions, fmt.Sprintf("%s=%s", actimeo, defaultActimeo))
+	}
+
+	if !mfsymlinksFlag {
```

Comment:
should it be `if mfsymlinksFlag` here?

Reply:
`if !mfsymlinksFlag` means user did not set that flag, then azure file driver should append that mount option.